modules/installation-generate-gcp-user-infra.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
```

Comment:
Seems like we'd want to merge this with `modules/installation-generate-aws-user-infra.adoc` into a generic `modules/installation-generate-user-infra.adoc` or some such to stay more DRY.

Reply:
It is so. Meet `modules/installation-user-infra-generate.adoc`. cc @kalexand-rh